script/Ciel/Day 10/10_01_CIEL10_4_2B.txt

```diff
@@ -89,204 +98,225 @@ If I had to transpose it to a Japanese context, maybe... Yes, think of it like a
 -- 「すみません、魔力でいいんですか？」
-"Sorry, is it 'magical energy' now?"
+"Sorry, but did you mean to say 'magical energy' there?"
 }
 [sha:0d9390f0aa0650034fb77ca62d4a02eef461cf48]{
 -- Page 213, Offset 27644.
 -- 「面倒なのでもういいです。
-"It's fine, I can't be bothered.
+"It's fine, I can't be bothered anymore.
 }
 [sha:43c7cddcb2d260293a8fd310649055b60a798c0b]{
 -- Page 213, Offset 27645.
 -- 聖堂教会は魔術協会と反目しているので呼称にはとてもうるさいのですが、どうでもいい問題なので」
-Since the Holy Church is in hostility with the Mage's Association, they're very picky about terminology, but it's an irrelevant matter."
+The Holy Church is at loggerheads with the Mages's Association,
+ so they're sticklers about the terminology, but it really doesn't matter."
 }
 [sha:5febab5804f412aede99f2f136197e45d5bc81fb]{
 -- Page 214, Offset 27646. Mods: @n.
 -- 「黒鍵は入手しやすい武器でもありましたが、使いにくい武器でもあったのです。
-"Black Keys were weapons that were easy to obtain, but difficult to use.
+"As weapons, Black Keys are easy to obtain but difficult to use.
 }
 [sha:78794dd5a80782d407da635d3db2c1ee0d8424fd]{
 -- Page 214, Offset 27647.
 -- 一人前の射手になるには10年の修業が必要なだけでなく、そもそもの構造がシンプルすぎてこれ以上の威力向上は望めなかった。
-One would need ten years of training to throw them competently, and even then, shouldn't expect any further improvement in potency because their structure was made from the start to be exceedingly simple.
+It takes ten years of training to be able to throw them competently,
+ and their intentionally oversimplified design meant there was no hope of further developing their power.
 }
 [sha:94fe77a781a1f6044410e44ccb4b09ab02acb1e6]{
 -- Page 214, Offset 27648.
 -- 年月を増すごとに強力になっていく死徒を相手にするには力不足だったのですね。
-It wasn't strong enough to fight against Dead Apostles, who become stronger with each passing year.
+They weren't enough to fend off the Dead Apostles, who became stronger with each passing year.
 }
 [sha:4e2adeb6b006e5a749a54a0da601ebc96a9da577]{
 -- Page 214, Offset 27649.
 -- 文明が発達し、銃器が量産できるようになって、黒鍵使いは数を減らしていきます」
-With the development of civilization and the beginning of the mass production of guns, the number of Black Keys users decreased."
+As civilization developed and the mass production of guns became widespread, the number of Black Key users decreased."
 }
 [sha:050d4d4451c0f1709ab6d0c7956666a6938e10f2]{
 -- Page 215, Offset 27650.
 -- 「1450年、グーテンベルク氏による印刷機の台頭で黒鍵の流行はちょっと持ち直しますが、やはり銃器のお手軽さには勝てず、黒鍵はメイン武装からサブ武装、『お守りがわりに持っていると通っぽい』ものに転落します。
-"With the emergence of Gutenberg's printing press, the popularity of Black Keys somewhat recovered, but not surprisingly, they weren't able to compete with the convenience of firearms. Black Keys went from being main weapons to sub-weapons, something that you would carry as a good luck charm if you were in the known.
+"Their popularity briefly recovered with the emergence of the Gutenberg press in 1450,
+ but they couldn't compete with the convenience of firearms.
+ Black keys went from being go-to weapons to an afterthought,
+ something you'd carry as a good-luck charm if you were in the know.
```

Comment:
Updated last sentence doesn't really come off with a clear meaning to me. Idea is that they're just carried as a good luck charm / accessory, right? Seems like just dropping `if you were in the know.` would be fine?

Reply:
The point of the sentence is that some people carried them around for basically ceremonial purposes because it made them look like they were more cultured. Kinda like how kings wore swords even after guns had been invented because it was a symbol of martial power.